### PR TITLE
Scheduled Access Review History Definition 

### PR DESCRIPTION
**Pull request by leherpel**
This is the documentation for scheduled access review history definitions.

Schema update PR: 5545147
Schema change PR to address [this review comment](https://github.com/microsoftgraph/microsoft-graph-docs/pull/15472#pullrequestreview-856453713): 5592092

### CODE REVIEW

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @leherpel for working on this docs PR. Shared a few suggestions for changes + the following gaps:
- Filename changes: accessreviewhistoryinstance-list.md => **accessreviewhistorydefinition-list-instances.md**
- Please surface the docs (accessReviewHistoryInstance.md and LIST method) and  in the TOC. You can nest this under the current accessReviewHistoryDefinition node.
- I see that ChangeList objects for the Deleted properties (fulfilledDateTime, downloadUri) were not generated. Please create them manually following the [changelog guidance](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog). The ChangeType is **Deletion**.
    - Suggestion for one of the change descriptions: `Deleted the **downloadUri** property from the [accessReviewHistoryDefinition](https://docs.microsoft.com/graph/api/resources/accessreviewhistorydefinition?view=graph-rest-beta) resource. Use the **downloadUri** property in the [accessReviewHistoryInstance](https://docs.microsoft.com/graph/api/resources/accessreviewhistoryinstance?view=graph-rest-beta) instead.`

**Additional comments:**

1. Ideally, we would have followed the deprecation guidance and first marked the two properties as deprecated then deleted them later. But I see in your schema PR that you deleted the 2 properties and the last property will be removed on January 30. Because of this, we can just go ahead and add the 2 changelist items for **Deletion** ChangeType.
2. I see your schema update PR is already merged into the master schema branch. Please prioritize these changes so that these docs can be published when schema deployment is done and the API goes live..

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 1]
```diff
@@ -663,6 +663,48 @@
...
       "SubArea": "Governance"
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "fe79cc68-75c6-47a3-95d9-7b1a3c772919",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessReviewHistoryScheduleSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the [accessReviewHistoryScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryScheduleSettings?view=graph-rest-beta) resource type.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:674]:
```suggestion
          "Description": "Added the [accessReviewHistoryScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewhistoryschedulesettings?view=graph-rest-beta) resource type. Use this resource to configure the settings of an [accessReviewHistoryDefinition](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewhistorydefinition?view=graph-rest-beta) object.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 2]
```diff
@@ -663,6 +663,48 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **scheduleSettings** property to [accessReviewHistoryDefinition](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryDefinition?view=graph-rest-beta) resource.",
+          "Target": "accessReviewHistoryDefinition"
+        },
+        {
+          "Id": "fe79cc68-75c6-47a3-95d9-7b1a3c772919",
+          "ApiChange": "Resource",
+          "ChangedApiName": "instances",
+          "ChangeType": "Addition",
+          "Description": "Added the **instances** relationship to [accessReviewHistoryDefinition](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryDefinition?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:690]:
```suggestion
          "Description": "Added the **instances** relationship to [accessReviewHistoryDefinition](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryDefinition?view=graph-rest-beta) resource. Represents the instances of a recurring acecss review history definition.",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
2 minor additions - enhancements to current changelog descriptions.

**Comment by leherpel** [comment]
@FaithOmbongi thank you so much for reviewing the PR. I updated the PR to reflect the changes you requested.

File: api-reference/beta/toc.yml [hunk 3]
```diff
@@ -12336,8 +12336,13 @@ items:
               href: api/accessreviewset-list-historydefinitions.md
             - name: Create
               href: api/accessreviewset-post-historydefinitions.md
-            - name: Generate download URI
-              href: api/accessreviewhistorydefinition-generatedownloaduri.md
+            - name: Access review history definition instance
```
Comment (FaithOmbongi) [inline api-reference/beta/toc.yml:12339]:
```suggestion
            - name: Access review history instance
```

File: api-reference/beta/resources/accessreviewhistorydefinition.md [hunk 4]
```diff
@@ -13,32 +13,41 @@ Namespace: microsoft.graph
...
-|reviewHistoryPeriodStartDateTime|DateTimeOffset|Timestamp, reviews starting on or before this date will be included in the fetched history data. Required.|
+|reviewHistoryPeriodEndDateTime|DateTimeOffset| A timestamp. Reviews ending on or before this date will be included in the fetched history data. Only required if **scheduleSettings** is not defined. |
+|reviewHistoryPeriodStartDateTime|DateTimeOffset|A timestamp. Reviews starting on or before this date will be included in the fetched history data. Only required if **scheduleSettings** is not defined.|
+| scheduleSettings  |[accessReviewHistoryScheduleSettings](accessReviewHistoryScheduleSettings.md)| The settings for a recurring access review history definition series. Only required if **reviewHistoryPeriodStartDateTime** or **reviewHistoryPeriodEndDateTime** are not defined.|
 |scopes|[accessReviewQueryScope](accessreviewqueryscope.md) collection|Used to scope what reviews are included in the fetched history data. Fetches reviews whose scope matches with this provided scope. Required.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewhistorydefinition.md:40]: To match the change in the JSON representation and the details in the schema
```suggestion
|scopes|[accessReviewScope](accessreviewscope.md) collection|Used to scope what reviews are included in the fetched history data. Fetches reviews whose scope matches with this provided scope. Required.|
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @leherpel, I see you changed the file name for the generateDownloadUri method from accessreviewhistorydefinition-generatedownloaduri.md => accessreviewhistorydefinition-generatedownloaduri-instance.md.

Looking at your schema, both the binding parameter and return type for generateDownloadUri is accessReviewHistoryDefinition. So the previous filename is/was correct.

However, I also see major changes you're making in the body of the doc that the path for the action is now under accessReviewHistoryInstance, does this mean that both the binding parameter and return type for generateDownloadUri is ~accessReviewHistoryDefinition~ accessReviewHistoryInstance? If so, then the file will change to **accessreviewhistoryinstance-generatedownloaduri.md**. But let's discuss this internally to confirm the right changes.

**Comment by leherpel** [comment]
Here is the PR to update the schema: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5591756

**Comment by leherpel** [comment]
Schema change: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5592092

**Review by FaithOmbongi — APPROVED** [review]
Thank you @leherpel for the quick work addressing the schema gap.

**Pull request merged**